docs/static/includes/interfaces/int.secExp.input.bicep

```diff
@@ -1,6 +1,6 @@
 secretsExportConfiguration: {
   keyVaultResourceId: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KeyVault/vaults/{keyVaultName}'
-  >secretToExportName1<: 'myPrimarySecret'
-  >secretToExportName2<: 'mySecondarySecret'
+  >secretToExport1<Name: 'myPrimarySecret'
```

Comment:
Lg(as for great)tm
Just if not too much effort, could we also fix the linting PR check error?

Reply:
done